src/test/java/apicurito/tests/steps/OperationSteps.java

```diff
@@ -98,16 +98,34 @@ public void setDescriptionForOverridePathParameterInOperation(String description
     public void setParametersTypes(DataTable table) {       //TODO  add specific parameter when it will be needed
 
         for (List<String> dataRow : table.cells()) {
-            String buttonId = CommonUtils.getButtonId(dataRow.get(0));
+            String buttonType = dataRow.get(0);
```

Comment:
The method is too complicated. Rather create a separate method for creating media types. Then it can be used for both Responses and Response Definition.